blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpEventsIngestionHandler.java

```diff
@@ -52,6 +55,23 @@ public HttpEventsIngestionHandler(EventsIO searchIO) {
 
     @Override
     public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
+
+        if ( !mediaTypeChecker.isContentTypeValid(request.headers()) ) {
+            DefaultHandler.sendResponse(ctx, request,
+                    String.format("Unsupported media type for Content-Type: %s", request.headers().get(HttpHeaders.Names.CONTENT_TYPE)),
+                    HttpResponseStatus.UNSUPPORTED_MEDIA_TYPE
+            );
+            return;
```

Comment:
The finally block is executing: 
```
DefaultHandler.sendResponse(ctx, request, response, status);
httpEventsIngestTimerContext.stop();
```
Both don't seem to be applicable in this case. The timer has not been started. And the response should be different.